tuta-sdk/rust/sdk/src/lib.rs

```diff
@@ -54,15 +63,21 @@ mod logging;
 pub mod login;
 mod mail_facade;
 mod metamodel;
+
+#[cfg(feature = "net")]
+pub mod net;
 pub mod rest_client;
 mod rest_error;
+pub mod services;
 mod simple_crypto;
-mod tutanota_constants;
+pub mod tutanota_constants;
 mod type_model_provider;
 mod typed_entity_client;
 mod user_facade;
 mod util;
 
+pub static CLIENT_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
we've been passing client version from the users of the SDK. Why is it necessary?

Reply:
I felt adding it to the bump-version script (forgot to do it in the end) and having a compile-time constant is better than passing around a string that could be anything. 
It's not like we're in any danger of following SemVer with this and the SDK should probably have some idea which client version it's supposed to be compatible with, even if looked at in isolation.

Reply:
I'm not opposed to it but let's make it a separate change then?

Reply:
Also, there are cases where we want app version number and there are cases where it's more protocol-related. I'm not sure if we maybe should split them somehow.

Reply:
it's a separate commit now. 

In my mind I guess the major version is the protocol-related part while the other parts of the version are more about which client you're using. I'm sure we could improve on the current headers to make it more independent.